Classes/ContentObject/JsonContentObject.php

```diff
@@ -167,7 +168,7 @@ protected function recursiveFind(array $haystack, $needle)
      * @param array $data
      * @return array
      */
-    protected function decodeFieldsIfRequired(array $data) : array
+    protected function decodeFieldsIfRequired(array $data): array
```

Comment:
FYI, I've added a rule to check this one in https://github.com/TYPO3-Initiatives/headless/pull/58